components/Header/index.js

```diff
@@ -1,5 +1,18 @@
+import { useRouter } from "next/router";
+
 function Header() {
-  return <h1>Bazam</h1>;
+  const { pathname } = useRouter();
+
+  console.log(pathname);
+  return (
+    <>
+      {pathname.startsWith("/articleDetails") ? (
+        <h1>Details</h1>
+      ) : (
+        <h1>Bazam</h1>
+      )}
```

Comment:
Just a little improvement suggestion here: I would recommend putting the first check logic over the `return` statement. Something like this:

```js
const isArticleDetails = {pathname.startsWith("/articleDetails")
return <h1>{isArticleDetails ? "Details" : "Bazam"}</h1>
````